bitnami/kafka/templates/_helpers.tpl

```diff
@@ -143,12 +143,31 @@ Return true if authentication via SASL should be configured for inter-broker com
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return true if encryption via TLS for client connections should be configured
+*/}}
+{{- define "kafka.client.tlsEncryption" -}}
+{{- $tlsProtocols := list "tls" "mtls" "sasl_tls" -}}
+{{- if (has .Values.auth.clientProtocol $tlsProtocols) -}}
+    {{- true -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return true if encryption via TLS for inter broker communication connections should be configured
+*/}}
+{{- define "kafka.interBroker.tlsEncryption" -}}
+{{- $tlsProtocols := list "tls" "mtls" "sasl_tls" -}}
+{{- if (has .Values.auth.interBrokerProtocol $tlsProtocols) -}}
+    {{- true -}}
+{{- end -}}
+{{- end -}}
+
```

Comment:
Instead of repeating code, I'd suggest using a function that received the value as a parameter:
```mustache
{{/*
Return true if variable "protocol" has a valid TLS protocol configured
Usage:
{{ include "kafka.tlsEnabled" ( dict "protocol" .Values.path.to.the.Value ) }}
*/}}
{{- define "kafka.tlsEnabled" -}}
{{- $saslProtocols := list "tls" "mtls" "sasl_tls" -}}
{{- if has .protocol $saslProtocols -}}
    {{- true -}}
{{- end -}}
{{- end -}}
```